templates/participants.html

```diff
@@ -68,3 +79,63 @@ <h1>Wszyscy uczestnicy</h1>
     </article>
 </div>
 {% endblock %}
+
+{% block script %}
+  <script>
+    $(document).ready(() => {
+      const table = $('#participants-table').DataTable({
```

Comment:
Being able to filter e.g. only qualified participants would also be nice

Reply:
Too much work - will be done in a separate PR.